src/cts/src/TritonCTS.cpp

```diff
@@ -575,11 +578,10 @@ std::vector<std::string> TritonCTS::findMatchingSubset(
   return subset;
 }
 
-bool TritonCTS::isClockBufferCandidate(sta::LibertyCell* buffer)
+bool TritonCTS::isClockCellCandidate(sta::LibertyCell* cell)
```

Comment:
What is the motivation to remove the isBuffer checking?

Reply:
This is to use the same check for both buffers and inverters.